e2e/cypress/tests/integration/enterprise/cloud/billing/notify_admin_spec.js

```diff
@@ -7,12 +7,31 @@
 // - Use element ID when selecting an element. Create one if none.
 // ***************************************************************
 
+function simulateFilesLimitReached(currentFileStorageUsageBytes, planLimit){
```

Comment:
Since the purpose of this function is only ever to set the usage above the limit, could we have this take a single arg, and the usage is set to that value, and the limit is set to that `value - 1`?